docs(tabs): 完善标签页组件的说明文档和用例 (#617)
TuSimple/naive-ui: src/tabs/demos/enUS/index.demo-entry.md

| Name | Type | Default | Description |
| --- | --- | --- | --- |
| addable | `boolean \| { disabled?: boolean }` | `false` | Whether to allow adding tags, it only takes effect when the tag's `type` is `card`. |
| closable | `boolean` | `false` | Whether to allow the tag to be closed, it only takes effect when the tag's `type` is `card`. |
| default-value | `string \| number` | `undefined` | Default value in uncontrolled mode. |

> | justify-content | `'space-between' \| 'space-around' \| 'space-evenly'` | `undefined` | `flex` layout. |

Reviewer: ```suggestion
| justify-content | `'space-between' \| 'space-around' \| 'space-evenly'` | `undefined` | Justify-content value of `flex` layout. |
```

| size | `'small' \| 'medium' \| 'large'` | `'medium'` | Size of tabs. |
| show-divider | `boolean` | `false` | Whether to show divider of tabs. Only works when `type` is `line`. |
| pane-style | `string \| object` | `undefined` | Style of the pane. |
| tab-style | `string \| object` | `undefined` | Style of the tab. |
| tabs-padding | `number` | `0` | Left & right `padding` of the group of tabs. |
| type | `'bar' \| 'line' \| 'card'` | `'bar'` | Tabs type. |

> | value | `string \| number` | `undefined` | Default value in controlled mode. |

Reviewer: ```suggestion
| value | `string \| number` | `undefined` | Value in controlled mode. |
```

> | on-add | `() => void` | `undefined` | Callback function for adding tags. |

Reviewer: ```suggestion
| on-add | `() => void` | `undefined` | Callback function triggered when add tag. |
```

> | on-close | `(name: string \| number) => void` | `undefined` | Callback function for deleting tags. |

Reviewer: ```suggestion
| on-close | `(name: string \| number) => void` | `undefined` | Callback function triggered when close tag. |
```

> | on-update:value | `(value: string \| number) => void` | `undefined` | Callback function when the selection changes. |

Reviewer: ```suggestion
| on-update:value | `(value: string \| number) => void` | `undefined` | Callback function triggered when the value changes. |
```

### Tab Pane Props

| Name | Type | Default | Description |
| --- | --- | --- | --- |

> | closable | `boolean` | `false` | Whether to allow the tag to be closed, it only takes effect when the tag's `type` is `card`. |

Reviewer: ```suggestion
| closable | `boolean` | `false` | Whether to allow the tag to be closed. Only works when the tag's `type` is `card`. |
```

| disabled | `boolean` | `false` | Whether to disable. |
| display-directive | `'if' \| 'show'` | `'if'` | The directive to use in conditionally rendering. `if` will use `v-if` and `show` will use `v-show`. When use show directive, the status of tab won't be reset after tab changes. |

> | tab | `string \| VNode \| () => VNodeChild` | `undefined` | Label display name. |

Reviewer: ```suggestion
| tab | `string \| VNode \| () => VNodeChild` | `undefined` | Tab label. |
```

> | name | `string \| number` | `undefined` | The name of the label. |

Reviewer: ```suggestion
| name | `string \| number` | `undefined` | Required, the name of the tab. |
```

## Slots

### Tabs Slots

| Name    | Parameters | Description   |
| ------- | ---------- | ------------- |
| default | `()`       | Tabs content. |
| prefix  | `()`       | Tabs prefix.  |
| suffix  | `()`       | Tabs suffix.  |

### Tab Pane Slots

| Name    | Parameters | Description              |
| ------- | ---------- | ------------------------ |
| default | `()`       | Tab content.             |

> | tab     | `()`       | What the tab item shows. |

Reviewer: ```suggestion
| tab     | `()`       | Tab label content. |
```
